packages/shared-components/src/components/Modal/Modal.js

```diff
@@ -63,9 +61,36 @@ class Modal extends React.Component {
      */
     Window: PropTypes.func,
     /**
-     * A component to render the close button of the modal
+     * A component to render the close icon in the right-top corner of the modal
+     */
+    CloseIcon: PropTypes.func,
+    /**
+     * Align action buttons
+     */
+    alignButtons: PropTypes.oneOf(['right', 'left', 'center']),
+    /**
+     * Space between buttons
+     */
+    buttonSpace: PropTypes.oneOf([
+      'xs',
+      'extraSmall',
+      'sm',
+      'small',
+      'md',
+      'medium',
+      'lg',
+      'large',
+      'xl',
+      'extraLarge',
+    ]),
+    /**
+     * Render Cancel button which on click will call onClose function
+     */
+    withCancelButton: PropTypes.bool,
+    /**
+     * Render Close button which on click will call onClose function
      */
-    CloseButton: PropTypes.func,
+    withCloseButton: PropTypes.bool,
```

Comment:
I'm trying to figure out if these two buttons are better served as something provided by the consumer of this library (as they are now) or automatically like this. I think maybe these shouldn't be included.